### PR TITLE
Jetty 12.0.x use automatic formatter for poms to have same style for every poms

### PR DESCRIPTION
**Pull request by olamy**
- setup spotless for poms only
- spotless:apply to all poms

### CODE REVIEW

**Comment by olamy** [comment]
issue https://github.com/eclipse/jetty.project/issues/6924

**Review by gregw — APPROVED** [review]

**Comment by joakime** [comment]
Looks cool.

Can this organize the dependency lists as well?
Like we do with our java source import lines?

**Comment by olamy** [comment]
@joakime yes dependencies sorting added, by `groupId, artifactId`

File: pom.xml [hunk 1]
```diff
@@ -5,64 +5,208 @@
...
+  <properties>
 
     <!-- dependency versions -->
     <alpn.agent.version>2.0.10</alpn.agent.version>
     <ant.version>1.10.14</ant.version>
     <apache.avro.version>1.11.2</apache.avro.version>
     <apache.httpclient.version>4.5.14</apache.httpclient.version>
     <apache.httpcore.version>4.4.16</apache.httpcore.version>
+
+    <!-- some maven plugins versions -->
```
Comment (joakime) [inline pom.xml:161]: We'll have to clean up some out of place (and no longer relevant) comments in a future PR

**Review by joakime — APPROVED** [review]
I like it, but lets talk with the rest of the group about when to merge this.

File: pom.xml (hunk 1 above)
Comment (olamy) [reply pom.xml:161]: sure. But I have to to fix the failure first .

**Pull request merged**